src/network/network-er-x-sfp.md

```diff
@@ -0,0 +1,351 @@
+ip route 100.127.2.252/30 dev l2tpeth63 table 10
+ip -6 route add fd74:6f6d:7368:7f02::fc/126 dev l2tpeth0 table 10
+```
+
+## OpenVPN - Management Tunnel
+
+Management tunnel to allow an alternative means of accessing the router out side of the mesh network, in case the mesh fails.
+
+Write certificates (provided by OpenVPN server)
+
+```
+cat <<"EOF">/config/auth/SN1R1.crt
+-----BEGIN CERTIFICATE-----
+<<SANITIZED>>
+-----END CERTIFICATE-----
+EOF
+cat <<"EOF">/config/auth/SN1R1.key
+-----BEGIN PRIVATE KEY-----
+<<SANITIZED>>
+-----END PRIVATE KEY-----
+EOF
+cat <<"EOF"> /config/auth/ca.crt
+-----BEGIN CERTIFICATE-----
+MIIEszCCA5ugAwIBAgIJAIiwPYS5mr9AMA0GCSqGSIb3DQEBCwUAMIGXMQswCQYD
+VQQGEwJDQTELMAkGA1UECBMCT04xEDAOBgNVBAcTB1Rvcm9udG8xFTATBgNVBAoT
+DFRvcm9udG8gTWVzaDEPMA0GA1UECxMGdG9tZXNoMQ8wDQYDVQQDEwZ0b21lc2gx
+DzANBgNVBCkTBnRvbWVzaDEfMB0GCSqGSIb3DQEJARYQaGVsbG9AdG9tZXNoLm5l
+dDAeFw0yMDA3MjAyMDEwNTNaFw0zMDA3MTgyMDEwNTNaMIGXMQswCQYDVQQGEwJD
+QTELMAkGA1UECBMCT04xEDAOBgNVBAcTB1Rvcm9udG8xFTATBgNVBAoTDFRvcm9u
+dG8gTWVzaDEPMA0GA1UECxMGdG9tZXNoMQ8wDQYDVQQDEwZ0b21lc2gxDzANBgNV
+BCkTBnRvbWVzaDEfMB0GCSqGSIb3DQEJARYQaGVsbG9AdG9tZXNoLm5ldDCCASIw
+DQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBAMkWwPiHTd9T1a82HXAJao771ZRJ
+1438yx6Ovm8mt+Z6MZ0r87BZ/pQ1TjcAgPNoV2TfnXkj8I1x2hoOub/EUu+WAYtf
+54Ozjse1nUEiXB8WHNTh1VsYqcDlna16Js9/jcK+WkQvolf9bHZgwzJxqV62Yt+P
+0pFPoIFsU4otdNCBKHmPF3UvEhq7Xa+BY3mtfFf6AIz/cjuS4sODz/jiNV9FfEbg
+0aW6/NOuM/TbUy7Cu3ubuf5ESMnI20jEBhln+ksCkzqLANegdMRp8d7WZxYkvdgb
+CYX9hRL83hQe2jhwijYw0DXvR5CqUwJXg8ERuRoxtpoNx00nuuk2dVY3gwUCAwEA
+AaOB/zCB/DAdBgNVHQ4EFgQUzuAyZEfBiPvL7oHQWAfkfMOC8O0wgcwGA1UdIwSB
+xDCBwYAUzuAyZEfBiPvL7oHQWAfkfMOC8O2hgZ2kgZowgZcxCzAJBgNVBAYTAkNB
+MQswCQYDVQQIEwJPTjEQMA4GA1UEBxMHVG9yb250bzEVMBMGA1UEChMMVG9yb250
+byBNZXNoMQ8wDQYDVQQLEwZ0b21lc2gxDzANBgNVBAMTBnRvbWVzaDEPMA0GA1UE
+KRMGdG9tZXNoMR8wHQYJKoZIhvcNAQkBFhBoZWxsb0B0b21lc2gubmV0ggkAiLA9
+hLmav0AwDAYDVR0TBAUwAwEB/zANBgkqhkiG9w0BAQsFAAOCAQEAjdLedME06tE3
+WZgZlA20TdlMThdV67hkKBZbGUhlIqJuaykSi2Fp8zVg5CxgGNOp5GJRQ2z1rMtr
+dPSIV6lYxfFgGTqfBRyw4dMyfDxA2XO3mMlTJzVNYcE1zXJp75ZpLxUEhcPNllYa
+l481UxG2SQdQh9UPrtKJxf3KSTvRJufzxrh6i3iPmZRzLhPt7Vc+aCYjUFq+EJ/z
+q1ACKn0pA8H3YbbdhA/RPQh3CRKiuE4kdq3BP4YYjW8xysrEU3TKfXHz4BY1OTPl
+JxRDuteL3nTPh22VfcY/j6OeVtGKg+BGJv4xRDEb0a0xx58Ki+9R6TPotNuHYj1L
+clmLw8d8Ew==
+-----END CERTIFICATE-----
+EOF
+```
+
+Configure OpenVPN named vtun99. 
+
+```
+set interfaces openvpn vtun99 mode client
+set interfaces openvpn vtun99 protocol udp
+set interfaces openvpn vtun99 remote-host 198.98.49.249
+set interfaces openvpn vtun99 remote-port 4141
+set interfaces openvpn vtun99 tls ca-cert-file /config/auth/ca.crt
+set interfaces openvpn vtun99 tls cert-file /config/auth/SN1R1.crt
+set interfaces openvpn vtun99 tls key-file /config/auth/SN1R1.key
+```
+
+If needed for the babeld workaround 2, a static route can be defined:
+
+```
+set protocols static route 199.195.250.208/32 next-hop 192.168.40.1
+```
+# Appendix A - Babeld - Cross-Compile
```

Comment:
Top level header here and an appendix to page seems odd. Leaving this up to you and @makeworld-the-better-one